tests/src/OneLogin/saml2_tests/utils_test.py

```diff
@@ -856,11 +856,15 @@ def testValidateSign(self):
         except Exception as e:
             self.assertEqual('Error parsing xml string', e.message)
 
+        self.assertRaises(Exception, OneLogin_Saml2_Utils.validate_sign, '', cert, raise_exceptions=True)
```

Comment:
Nit: I strongly prefer the context manager syntax:
```python
with self.assertRaises(Exception):
    OneLogin_Saml2_Utils.validate_sign('', cert, raise_exceptions=True)
```
or, even better, combine that with checking the specific message:
```python
with self.assertRaisesRegexp(Exception, "Empty string supplied as input"):
    OneLogin_Saml2_Utils.validate_sign('', cert, raise_exceptions=True)
```